### ISSUE TITLE
Error during initialization Angular2.RC4

### ISSUE THREAD

**Issue by moohkooh** [Gbuomprisco/ng2-tag-input#38]
i adding the tag-input into my application. On init i getting an error:

*Can't bind to '@fade' since it isn't a known native property.* 
Im running on Angular2 RC4.
My HTML is simple (copied from example page)
```javascript
<tag-input [ngModel]="['@item']"
      [onlyFromAutocomplete]="true"
      [autocompleteItems]="['item1', 'item2']"
      [autocomplete]="true">
 </tag-input>
```

My Component:
```javascript
@Component({
    selector: 'test-tag',
    templateUrl: 'tags.html', 
    directives: [TagInput],
    providers: [NG_FORM_PROVIDER]
})
```

Error:
```
Can't bind to '@fade' since it isn't a known native property ("

    <!-- MENU -->
    <div class='ng2-dropdown-menu' [ERROR ->][@fade]="state.toString()">
        <ng-content></ng-content>
    </div>
"): Ng2DropdownMenu@6:35 
```

**Comment by Gbuomprisco** [comment]
Hi @moohkooh!

The latest version of this component is only compatible with Angular RC.5. If you want to use it with the RC.4 you may want to downgrade it to the version 0.2.2/0.2.1. 

I obviously recommend to upgrade to Angular 2 to RC.5 :)

**Comment by moohkooh** [comment]
make sense 👍  Today i upgrade to RC.5. I needed to add a fromGroupName as attribute, but on autocomplete i getting an exception and the tags are on the middle of the control (is there maybe a problem with bootstrap?)

 1. Autocomplete error msg:

```
platform-browser.umd.js:937 TypeError: Cannot read property 'state' of undefined
    at Ng2DropdownMenu.handleKeypress (ng2-dropdown-menu.ts:106)
    at eval (platform-browser.umd.js:1836)
    at eval (platform-browser.umd.js:1913)
    at ZoneDelegate.invoke (zone.js?1472024685631:332)
    at Object.onInvoke (core.umd.js:9245)
    at ZoneDelegate.invoke (zone.js?1472024685631:331)
    at Zone.runGuarded (zone.js?1472024685631:239)
    at NgZoneImpl.runInnerGuarded (core.umd.js:9278)
    at NgZone.runGuarded (core.umd.js:9510)
    at HTMLBodyElement.outsideHandler (platform-browser.umd.js:1913)
```

2. Screenshot: https://i.imgsafe.org/d51b04e6e1.png

**Comment by Gbuomprisco** [comment]
@moohkooh that's interesting. Sorry, I did not understand if you did update or you are going to do it? :)

Can you paste here your package.json dependencies? I'll be able to understand a little more. 

Thanks!

**Comment by moohkooh** [comment]
I upgraded my app to Angular2.RC5 :)

my package.json
```
"devDependencies": {
    "async": "^2.0.0",
    "autoprefixer": "^6.3.7",
    "browser-sync": "^2.13.0",
    "codelyzer": "0.0.25",
    "colorguard": "^1.2.0",
    "compression": "^1.6.2",
    "connect": "^3.4.1",
    "connect-history-api-fallback": "^1.2.0",
    "connect-livereload": "^0.5.4",
    "cssnano": "^3.7.3",
    "deep-extend": "^0.4.1",
    "doiuse": "^2.4.1",
    "event-stream": "^3.3.3",
    "express": "~4.14.0",
    "express-history-api-fallback": "^2.0.0",
    "extend": "^3.0.0",
    "gulp": "^3.9.1",
    "gulp-cached": "^1.1.0",
    "gulp-clean-css": "^2.0.11",
    "gulp-concat": "^2.6.0",
    "gulp-concat-css": "^2.3.0",
    "gulp-debug": "^2.1.2",
    "gulp-filter": "^4.0.0",
    "gulp-inject": "^4.1.0",
    "gulp-inline-ng2-template": "^2.0.5",
    "gulp-load-plugins": "^1.2.4",
    "gulp-jsoncombine": "^1.0.3",
    "gulp-plumber": "~1.1.0",
    "gulp-postcss": "^6.1.1",
    "gulp-progeny": "^0.2.1",
    "gulp-protractor": "^2.3.0",
    "gulp-replace": "^0.5.4",
    "gulp-sass": "^2.3.1",
    "gulp-sass-lint": "^1.1.1",
    "gulp-shell": "~0.5.2",
    "gulp-sourcemaps": "2.0.0-alpha",
    "gulp-template": "^4.0.0",
    "gulp-tslint": "^6.0.1",
    "gulp-typescript": "~2.13.6",
    "gulp-uglify": "^1.5.4",
    "gulp-util": "^3.0.7",
    "gulp-watch": "^4.3.8",
    "is-ci": "^1.0.8",
    "protractor": "^4.0.0",
    "isstream": "^0.1.2",
    "jasmine-core": "~2.4.1",
    "jasmine-spec-reporter": "^2.5.0",
    "jasmine-reporters": ">=2.0.0",
    "karma": "~0.13.22",
    "karma-chrome-launcher": "~1.0.1",
    "karma-coverage": "^1.1.0",
    "karma-jasmine": "~1.0.2",
    "karma-junit-reporter": "^1.0.0",
    "karma-mocha-reporter": "^2.0.4",
    "karma-phantomjs-launcher": "^1.0.0",
    "merge-stream": "^1.0.0",
    "ng2lint": "0.0.10",
    "node-sass": "3.7.0",
    "protractor-jasmine2-html-reporter": "^0.0.6",
    "open": "0.0.5",
    "phantomjs-prebuilt": "^2.1.4",
    "postcss-reporter": "^1.4.1",
    "remap-istanbul": "^0.6.4",
    "rimraf": "^2.5.3",
    "run-sequence": "^1.2.2",
    "semver": "^5.3.0",
    "serve-static": "^1.11.1",
    "slash": "~1.0.0",
    "stream-series": "^0.1.1",
    "stylelint": "^7.0.2",
    "stylelint-config-standard": "^11.0.0",
    "systemjs-builder": "0.15.14",
    "tildify": "^1.2.0",
    "tiny-lr": "^0.2.1",
    "traceur": "^0.0.111",
    "ts-node": "^1.0.0",
    "tslint": "^3.13.0",
    "tslint-stylish": "2.1.0-beta",
    "typescript": "~1.8.10",
    "typings": "^1.3.1",
    "vinyl-buffer": "^1.0.0",
    "vinyl-source-stream": "^1.1.0",
    "walk": "^2.3.9",
    "yargs": "^4.8.0"
  },
  "dependencies": {
    "@angular/common": "2.0.0-rc.5",
    "@angular/compiler": "2.0.0-rc.5",
    "@angular/core": "2.0.0-rc.5",
    "@angular/forms": "0.3.0",
    "@angular/http": "2.0.0-rc.5",
    "@angular/platform-browser": "2.0.0-rc.5",
    "@angular/platform-browser-dynamic": "2.0.0-rc.5",
    "@angular/router": "3.0.0-rc.1",
    "bignumber": "^1.1.0",
    "bignumber.js": "^2.3.0",
    "blob-stream": "^0.1.3",
    "bootstrap": "^3.3.6",
    "bootstrap-sass": "^3.3.6",
    "bourbon": "^4.2.6",
    "cbor": "^1.0.1",
    "core-js": "^2.4.0",
    "dexie": "1.3.4",
    "es6-module-loader": "^0.17.8",
    "ev-emitter": "^1.0.2",
    "gulp-sass": "^2.2.0",
    "jquery": "^2.2.3",
    "jqueryui": "^1.11.1",
    "lodash": "^4.14.1",
    "mfb": "^0.12.0",
    "moment": "^2.13.0",
    "ng2-tag-input": "^0.2.3",
    "pdfkit": "^0.7.2",
    "reflect-metadata": "0.1.2",
    "rxjs": "5.0.0-beta.6",
    "screenfull": "^3.0.0",
    "snapsvg": "^0.4.0",
    "spinkit": "^1.2.5",
    "systemjs": "0.19.27",
    "toastr": "^2.1.2",
    "whatwg-fetch": "^1.0.0",
    "zone.js": "^0.6.12"
  }
```

**Comment by Gbuomprisco** [comment]
Apparently this is not related to the upgrade. Can you post some of your code(template and controller)?

**Comment by Gbuomprisco** [comment]
Hi @moohkooh, I just published the version 0.2.4. It should fix this, let me know otherwise :)
